client/src/components/main.js

```diff
@@ -80,55 +80,96 @@ class Main extends Component {
   }
 
   onAcceptCookie() {
-    BrowserCookies.set('cookie-advice-accepted', "yes", {expires: 365});
+    BrowserCookies.set('cookie-advice-accepted', 'yes', { expires: 365 });
```

Comment:
Here there are changes in the quote and spacing around brackets. As they are not related to the issue, they shouldn't be here either. Any changes in style of pre-existing code should be addressed in another PR.